src/kartograf/tests/test_atom_mapper.py

```diff
@@ -287,4 +289,26 @@ def test_partial_fused_rings(fused_ring_mols, allow_partial_fused_rings, expecte
             fused_ring_mols[1],
         )
     )
-    check_mapping_vs_expected(mapping=geom_mapping, expected_mapping=expected_mapping)
\ No newline at end of file
+    check_mapping_vs_expected(mapping=geom_mapping, expected_mapping=expected_mapping)
+
+
+def test_hybridization_and_ring_breaks(shp2_hybridization_ligands):
+    """
+    Make sure rings are not broken when map_exact_ring_matches_only=True and a custom filter is used.
+    """
+    mapper = KartografAtomMapper(
+        map_exact_ring_matches_only=True,
+        additional_mapping_filter_functions=[filter_hybridization_changes]
```

Comment:
What would happen if you include the default filters in the additional mapping filters, would it skip them/still preserve the correct order?
E.g.
```
additional_mapping_filter_functions=[
                filter_ringbreak_changes, # default
                filter_ringsize_changes, # default
                filter_whole_rings_only, # default
                filter_hybridization_changes]
```

Reply:
We currently don't check what filters are passed as additional so if a default filter was used then it would be applied twice first as an additional filter then as the default, maybe I should expand the docs to make it clear that defaults are applied first?

Reply:
I think a warning would be fair we don't want to error out and stop experimentation and a section in the docs which explains exactly which filters are applied when you set a default filter should help.